pharmacy_retail/demo/pos_config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="point_of_sale.pos_config_main" model="pos.config">
```

Comment:
This one should be archived (in demo) and we should define a new `pos.config` in data, as done in other modules.

Reply:
okay